core/domain/rights_manager.py

```diff
@@ -753,7 +774,15 @@ def _assign_role(
     committer_id = committer.user_id
     activity_rights = _get_activity_rights(activity_type, activity_id)
 
-    if not check_can_modify_activity_roles(committer, activity_rights):
+    role_is_voice_artist = new_role == constants.ROLE_VOICE_ARTIST
```

Comment:
Maybe put parens around `new_role == constants.ROLE_VOICE_ARTIST` for readability.